web/gds-admin-ui/src/components/ResendEmail.js

```diff
@@ -96,7 +99,7 @@ function ResendEmail() {
             reason: ""
         }
         setIsSubmitting(true)
-        axios.post(`/vasps/${vasp?.id}/resend`, payload, {
+        api.create(`/vasps/${vasp?.id}/resend`, payload, {
```

Comment:
@elysee - this makes sense that this would fix the issue; I'm assuming that it wasn't correctly redirecting to `/v2` before. 

Before we merge this in, could you remove the `gds-user-ui` assets or rebase as needed?

Reply:
sure